.jenkins/pipelines/Azure/Windows/Jenkinsfile

```diff
@@ -77,6 +77,29 @@ def windowsCrossCompile(String label, String build_type, String lvi_mitigation =
     }
 }
 
+def windowsCrossPlatform(String label) {
+    def node_label = AGENTS_LABELS[label]
+    stage("Windows ${node_label} Cross Plaform Build") {
+        node(node_label) {
+            timeout(GLOBAL_TIMEOUT_MINUTES) {
+                cleanWs()
+                checkout scm
+
+                dir("devex/cross-nuget/standalone-builds/windows") {
+                    bat(
+                        returnStdout: false,
+                        returnStatus: false,
+                        script: """
+                            vcvars64.bat x64 && powershell -c "Set-ExecutionPolicy Bypass -Scope Process; .\\build.ps1 -SDK_PATH ${WORKSPACE}"
+                            vcvars64.bat x64 && powershell -c "Set-ExecutionPolicy Bypass -Scope Process; .\\pack.ps1"
```

Comment:
Out of curiosity, does this catch any errors that happen in the powershell scripts?

Reply:
From what I've seen before, it should